tests/test_parsers.py

```diff
@@ -3,34 +3,42 @@
 import pathlib
 import unittest
 
-from bsb.config.parsers import ConfigurationParser, get_configuration_parser
+from bsb.config.parsers import (
+    ConfigurationParser,
+    ParsesReferences,
+    get_configuration_parser,
+)
 from bsb.exceptions import ConfigurationWarning, FileReferenceError, PluginError
 
 
 def get_content(file: str):
     return (pathlib.Path(__file__).parent / "data/configs" / file).read_text()
 
 
-class RefParserMock(ConfigurationParser):
+class RefParserMock(ConfigurationParser, ParsesReferences):
```

Comment:
mixins typically come before the base class in the MRO

```suggestion
class RefParserMock(ParsesReferences, ConfigurationParser):
```